apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -305,19 +335,18 @@ block_by_height(_Config) ->
     ok.
 
 block_not_found_by_height(_Config) ->
-    TopHeader = rpc(aec_conductor, top_header, []), 
-    0 = aec_headers:height(TopHeader), %chain is empty
-    NumberOfChecks = 10,
+    InitialHeight = aec_blocks:height(rpc(aec_conductor, top, [])),
+    NumberOfChecks = 5,
     lists:foreach(
         fun(_) ->
-            Height = rand:uniform(99) + 1, % random number 1-100
+            Height = rand:uniform(99) + 1 + InitialHeight, % random number 1-100
```

Comment:
I am concerned that while tests executes from `aec_conductor:top` to here, the node managed to find one ore more blocks (unlikely but possible). Would it make sense to add a padding of e.g. 2 blocks? (out of scope of this PR of course)

Reply:
The node mining a new block is quite unlikely since there is no ongoing mining attempt. Since this is an expectation for the test,  I'll add an appropriate assertion.